src/global-firestore-data/toggleable-requirement-choices.ts

```diff
@@ -0,0 +1,19 @@
+import { toggleableRequirementChoicesCollection } from '@/firebase-frontend-config';
+import store from '@/store';
+
+const editToggleableRequirementChoices = (
+  toggleableRequirementChoices: AppToggleableRequirementChoices
+): void => {
+  toggleableRequirementChoicesCollection
+    .doc(store.state.currentFirebaseUser.email)
+    .set(toggleableRequirementChoices);
+};
+
+const chooseToggleableRequirementOption = (requirementID: string, option: string): void => {
+  editToggleableRequirementChoices({
+    ...store.state.toggleableRequirementChoices,
+    [requirementID]: option,
+  });
+};
```

Comment:
I split up `chooseToggleableRequirementOption` into two functions in order to encapsulate `editToggleableRequirementChoices` (for which the functionality was previously exposed).